assignments/callbacks.js

```diff
@@ -70,11 +70,24 @@ multiplyNums(3, 3, function (multiply) {
   console.log(multiply);
 });
 
+
 //Exercise 5
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
-  // Pass true to the callback if it is, otherwise pass false.
-}
+  // Pass true to the callback if it is, otherwise pass false
+  for (let i = 0; i < list.length; i++) {
+    if (list[i] === item) {
+      return cb(true);
+    } else {
+      return cb(false);
+    }
+  };
+};
+
+contains('Apple', items, function (containsItem) {
+  console.log(containsItem);
+});
+
```

Comment:
CB's look good. I believe you can use the includes method if you want get rid of the for loop.